vagrant/README.md

```diff
@@ -6,3 +8,16 @@ In order to disable the rsync option in your Vagrantfile, you need to use this s
 config.vm.synced_folder ".", "/home/vagrant/sync", disabled: true
 ~~~
 
+##Setup VirtualBox shared folder at /vagrant
+
+Install vagrant-vbguest
```

Comment:
Even better if this is something that can be included in the published builds. I don't know enough about building boxes though. My thinking was just to save people from googling this for two hours after downloading the box from Atlas.